orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -885,23 +913,60 @@ void addObjectMethodsForTraining(py::module& m, ExecutionProviderRegistrationFn
       .def(py::init([](const std::string& model_uri,
                        onnxruntime::training::api::CheckpointState* state,
                        std::optional<std::string> eval_model_uri,
-                       OrtDevice device) {
-        onnxruntime::SessionOptions session_option;
+                       OrtDevice device, PySessionOptions* session_options) {
         std::vector<std::shared_ptr<IExecutionProvider>> provider = GetExecutionProvidersForTrainingApis(device);
-
         auto env = GetTrainingEnv().GetORTEnv();
-        return std::make_unique<onnxruntime::training::api::Module>(
-            model_uri, state, session_option, *env, provider, eval_model_uri);
+        auto model = std::make_unique<onnxruntime::training::api::Module>(
```

Comment:
nit: why not just return without assignment to local varl as before?

Reply:
Thanks. I was playing around locally and forgot to revert back later.